spec/paths_spec.rb

```diff
@@ -42,4 +42,25 @@
       end
     end
   end
+
+  describe "dnsmasq_conf" do
```

Comment:
Minor, but we could simplify this test by stubbing the `govuk_docker_dir` it delegates to.